packages/itinerary-body/src/TransitLegBody/alerts-body.tsx

```diff
@@ -106,34 +108,35 @@ export default function AlertsBody({
                 )}
                 <S.TransitAlertBody>{description}</S.TransitAlertBody>
                 <S.TransitAlertEffectiveDate>
-                  {Math.abs(dayDiff) <= 1 ? (
-                    <FormattedMessage
-                      defaultMessage={
-                        defaultMessages[
-                          "otpUi.TransitLegBody.AlertsBody.effectiveTimeAndDate"
-                        ]
-                      }
-                      description="Text with the time and date an alert takes effect"
-                      id="otpUi.TransitLegBody.AlertsBody.effectiveTimeAndDate"
-                      values={{
-                        dateTime: effectiveStartDate * 1000,
-                        day: <AlertDay dayDiff={dayDiff} />
-                      }}
-                    />
-                  ) : (
-                    <FormattedMessage
-                      defaultMessage={
-                        defaultMessages[
-                          "otpUi.TransitLegBody.AlertsBody.effectiveDate"
-                        ]
-                      }
-                      description="Text with the date an alert takes effect"
-                      id="otpUi.TransitLegBody.AlertsBody.effectiveDate"
-                      values={{
-                        dateTime: effectiveStartDate * 1000
-                      }}
-                    />
-                  )}
+                  {showAlertEffectiveDateTimeText &&
+                    (Math.abs(dayDiff) <= 1 ? (
```

Comment:
What happens if you wrap the parent component `S.TransitAlertEffectiveDate` with the new boolean? I think this would make it a bit clearer just because it would break up this bit conditional block here that can be a bit hard to follow. At first I thought it was `showAlertEffectiveDateTimeText && Math.abs(dayDiff) <= 1` because I didn't see the `(` so I think breaking it up would be good just for clarity.

Reply:
addressed in dc9fb59